.github/workflows/deploy.yml

```diff
@@ -27,6 +27,11 @@ jobs:
       with:
         java-version: 11
 
+    - name: Get the lastest Databricks JDBC driver
+      run: |
+        curl -k https://databricks-bi-artifacts.s3.us-east-2.amazonaws.com/simbaspark-drivers/jdbc/2.6.22/SimbaSparkJDBC42-2.6.22.1040.zip --output SparkJDBC.jar.zip
```

Comment:
TODO: remove the `-k`.  Previously the certificate was self-signed, but appears to now be fixed.